src/DateUtils.ts

```diff
@@ -16,6 +16,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+import { Optional } from "matrix-events-sdk";
```

Comment:
Not related to this PR: Is that probably something we should move to somewhere else? Feels wrong to depend on the events-sdk just to get a utility type.